android/src/main/java/com/stario10module/PrinterBuilderWrapper.kt

```
@@ -4,386 +4,52 @@ import com.facebook.react.bridge.*
+                    )
+            )
+
+            val jsonObject = JSONObject(builder.getCommands())
+
+            val contents = jsonObject
+                .getJSONArray("contents").getJSONObject(0)
+                .getJSONArray("contents").getJSONObject(0)
+                .getJSONArray("contents").getJSONObject(0)
+
+            promise.resolve(contents.toString())
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)